interfaces/PA-App/src/app/services/padata.service.ts

```diff
@@ -114,11 +114,7 @@ export class PaDataService {
 
     const paAccountApiOutput = await this.paAccountApi.retrieve(type);
 
-    // Only JSON.parse the output if it could result in something:
-    if (!paAccountApiOutput) {
-      return undefined;
-    }
-    return JSON.parse(paAccountApiOutput);
+    return paAccountApiOutput;
```

Comment:
We 'trust' the API-handler to do everything for us and only return sane results...